setup.py

```diff
@@ -55,15 +55,15 @@ def get_requirements(requirements_filename: str):
                 version = line.split("'")[1]
 
 setup(
-    name='neon-tts-plugin-mozilla_remote',
+    name='neon-tts-plugin-mozilla-remote',
```

Comment:
Is this change necessary? It seems that it would create a new package on PyPI

Reply:
this was just to keep the naming convention in plugins + the convention of matching plugin name to package name

but can be reverted to not create a new package, those are just guidelines not required!

Reply:
Okay, let's revert this one and try to keep up the standards in the future